src/lib/theme/light/variants/buttons.ts

```diff
@@ -1,78 +1,143 @@
 import { mergeDeepRight } from 'ramda';
 
-import { space } from '../../design-system/space';
 import { button } from '../../variants';
 import { colors } from '../colors';
 
 const defaultButton = mergeDeepRight(button, {
   ':disabled': {
-    background: colors['container'],
+    background: '#00000029',
```

Comment:
You can add the specific color variable if exists

Reply:
this Colors dosent exist mapped on Theme it's the reason that i put this way...